libs/environment/src/components/node-switcher-dialog/node-switcher-dialog.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ComponentProps } from 'react';
+import { Dialog } from '@vegaprotocol/ui-toolkit';
+import { NodeSwitcher } from '../node-switcher';
+
+type NodeSwitcherDialogProps = ComponentProps<typeof NodeSwitcher> & {
+  dialogOpen: boolean;
+  setDialogOpen: (dialogOpen: boolean) => void;
+};
+
+export const NodeSwitcherDialog = ({
+  config,
+  dialogOpen,
+  setDialogOpen,
+  onConnect,
+}: NodeSwitcherDialogProps) => {
+  return (
+    <Dialog open={dialogOpen} contentClassNames="md:w-[856px] w-[856px]">
```

Comment:
hmm, this is a tricky one. 

the dialog seems to have a set max width of 520px, which doesn't support this element I need to put into it:
<img width="630" alt="Screenshot 2022-07-11 at 15 50 58" src="https://user-images.githubusercontent.com/105208209/178292877-7be19a26-2051-4674-bd29-75f0d4d21a82.png">

there was no consistent width for the node switcher in the designs, so i went with a pretty arbitrary width here which I thought it doesn't break the layout we want to show.

if we were going to make widths reusable, do we use any patterns in the design system which would make it obvious which other elements this might match? it seems like a bit of an overkill to try to go over the whole design system trying to match it with something else. spacing in general, paddings and margins I can see how they are related to a uniform image of a brand, dialog widths for custom data displays, not sure.

Reply:
Ah I see, thanks @notbot00. I spoke with Pete and we agreed 520px is too small for most use cases. Can you set it to default to 1000px and remove the 856px from here please?

Reply:
awesome, thanks for checking! fixed this by making the dialog size 1000px on screens > md.